tests/zendesk/test_zendesk.py

```diff
@@ -0,0 +1,144 @@
+                  "sharing_agreements", "skips", "suspended_tickets", "targets", "ticket_forms", "ticket_metrics", "triggers", "user_fields", "views", "tags"]
+CHAT_TABLES = ["chats"]
+# calls
+TALK_TABLES = ["calls", "addresses", "agents_activity", "current_queue_activity", "greeting_categories", "greetings", "ivrs", "lines", "phone_numbers", "settings", "calls_incremental",
+               "legs_incremental"]
+# all the timezones saved in dlt state
+INCREMENTAL_SAVED_KEYS = ["last_load_tickets", "last_load_ticket_metric_events", "last_load_chats", "last_load_talk_calls", "last_load_talk_legs"]
+
+
+@pytest.mark.parametrize("destination_name", ALL_DESTINATIONS)
+def test_pivoting_tickets(destination_name: str) -> None:
+    """
+    Checks to see that the tickets table uses pivoting
+    """
+
+    # Run pipeline without pivoting - get columns of tickets and check that it has the expected columns
+    pipeline_pivoting_1 = dlt.pipeline(destination=destination_name, full_refresh=True, dataset_name="test_unpivot_tickets_support")
+    data = zendesk_support(load_all=False, pivot_ticket_fields=False)
+    info = pipeline_pivoting_1.run(data.with_resources("ticket_fields", "tickets"))
+    assert_load_info(info)
+    schema = pipeline_pivoting_1.default_schema
+    unpivoted_tickets = schema.all_tables()[1]["columns"].keys()
+    assert "custom_fields" in unpivoted_tickets
+    assert "test_field" not in unpivoted_tickets
+
+    # run pipeline with pivoting - get columns of tickets and check that it has the expected columns
+    pipeline_pivoting_2 = dlt.pipeline(destination=destination_name, full_refresh=True, dataset_name="test_pivot_tickets_support")
+    data2 = zendesk_support(load_all=False, pivot_ticket_fields=True)
+    info2 = pipeline_pivoting_2.run(data2.with_resources("ticket_fields", "tickets"))
+    assert_load_info(info2)
+    schema2 = pipeline_pivoting_2.default_schema
+    pivoted_tickets = schema2.all_tables()[1]["columns"].keys()
+    assert "test_field" in pivoted_tickets
+    assert "custom_field" not in pivoted_tickets
+
+
+@pytest.mark.parametrize("destination_name", ALL_DESTINATIONS)
+def test_incrementing(destination_name: str) -> None:
+    """
+    Checks that incremental loading works for ZendeskSupport
+    @:param: destination_name - redshift/bigquery/postgres
+    """
+
+    # run pipeline
+    pipeline_incremental = dlt.pipeline(destination=destination_name, full_refresh=True, dataset_name="test_incremental")
+    data_support = zendesk_support(load_all=False)
+    data_chat = zendesk_chat()
+    data_talk = zendesk_talk()
+    info_incremental = pipeline_incremental.run([data_support, data_chat, data_talk])
+    # check that the expected keys are saved now in dlt state
+    for saved_timezone in INCREMENTAL_SAVED_KEYS:
+        assert isinstance(dlt.state()["zendesk"][saved_timezone], float)
+    assert_load_info(info_incremental, expected_load_packages=3)
```

Comment:
at least load it again and check if there are no additional records.
see github for similar test:
https://github.com/dlt-hub/pipelines/blob/master/tests/github/test_github_pipeline.py#L36

and load_table_counts and load_table_distinct_counts that will help you to test against duplicates